<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SunsetPluginSettingsPaneClass</class>
 <widget class="QWidget" name="SunsetPluginSettingsPaneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestPluginSettingsPane</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>&amp;Dateiname:</string>
     </property>
     <property name="buddy">
      <cstring>filenameInput</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="filenameInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="findFileButton">
       <property name="text">
        <string>Durch&amp;suchen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Der Dateiname kann ohne Verzeichnis angegeben werden,
sofern sich die Datei im Plugin-Pfad befindet.</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Quelle:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="sourceLabel">
     <property name="text">
      <string>unbekannt</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Datei:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="filenameLabel">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>filenameInput</tabstop>
  <tabstop>findFileButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
